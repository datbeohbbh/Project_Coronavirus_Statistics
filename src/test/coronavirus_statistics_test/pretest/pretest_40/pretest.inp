50 100
q 1 16 367
? 4 1
q 1 29 222
? 4 2
q 1 27 238
? 4 1
q 3 16 113
? 4 3
q 2 16 7
q 1 4 335
? 4 2
q 1 16 54
? 4 2
q 2 27 185
? 4 3
q 1 28 295
q 1 23 387
q 2 29 107
q 3 4 218
? 7
? 6
? 6
q 3 27 33
q 2 16 231
q 1 12 94
? 4 3
q 3 29 85
q 1 7 104
q 3 27 7
q 1 23 256
q 1 20 335
q 2 7 72
q 2 23 494
q 1 23 28
? 4 9
? 4 4
q 2 4 53
q 1 35 45
q 1 9 9
? 7
q 1 13 238
q 2 29 29
? 4 1
? 4 1
? 7
? 6
? 4 5
q 2 23 18
? 7
? 4 9
q 2 16 15
q 3 13 111
q 2 28 70
q 2 9 6
? 4 9
q 1 25 491
? 7
q 1 26 285
q 2 7 26
q 2 12 21
q 3 16 21
q 2 27 7
? 6
? 4 2
q 1 30 207
q 1 32 159
? 6
? 4 9
q 2 35 6
q 3 28 6
q 1 39 187
q 2 29 1
? 4 7
? 4 4
? 4 10
q 1 47 408
q 1 44 77
? 4 18
? 4 8
? 4 4
q 2 13 98
q 2 26 198
? 4 14
? 4 9
? 4 1
? 4 18
? 7
? 7
? 4 7
? 7
? 4 14
? 7
? 4 16
? 7
? 6
? 7
? 4 10
? 4 2
? 4 5
? 7