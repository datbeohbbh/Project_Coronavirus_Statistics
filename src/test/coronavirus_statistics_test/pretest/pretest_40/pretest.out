Query ? 4 1 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 367
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 367
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 29 
- Information of the city 29 -
-> Number of current patients : 222
-> Has recovered : 0
-> Was dead : 0

Query ? 4 3 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 254
-> Has recovered : 0
-> Was dead : 113

Query ? 4 2 :
The answer is the city with the index 27 
- Information of the city 27 -
-> Number of current patients : 238
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 27 
- Information of the city 27 -
-> Number of current patients : 238
-> Has recovered : 0
-> Was dead : 0

Query ? 4 3 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 301
-> Has recovered : 7
-> Was dead : 113

Query ? 7 :
The city with the highest death is city with the index 4
Quantity: 218 people

Query ? 6 :
The city with the highest recovery is city with the index 27
Quantity: 185 people

Query ? 6 :
The city with the highest recovery is city with the index 27
Quantity: 185 people

Query ? 4 3 :
The answer is the city with the index 12 
- Information of the city 12 -
-> Number of current patients : 94
-> Has recovered : 0
-> Was dead : 0

Query ? 4 9 :
The answer is the city with the index 20 
- Information of the city 20 -
-> Number of current patients : 335
-> Has recovered : 0
-> Was dead : 0

Query ? 4 4 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 70
-> Has recovered : 238
-> Was dead : 113

Query ? 7 :
The city with the highest death is city with the index 4
Quantity: 218 people

Query ? 4 1 :
The answer is the city with the index 29 
- Information of the city 29 -
-> Number of current patients : 1
-> Has recovered : 136
-> Was dead : 85

Query ? 4 1 :
The answer is the city with the index 29 
- Information of the city 29 -
-> Number of current patients : 1
-> Has recovered : 136
-> Was dead : 85

Query ? 7 :
The city with the highest death is city with the index 4
Quantity: 218 people

Query ? 6 :
The city with the highest recovery is city with the index 23
Quantity: 494 people

Query ? 4 5 :
The answer is the city with the index 35 
- Information of the city 35 -
-> Number of current patients : 45
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 4
Quantity: 218 people

Query ? 4 9 :
The answer is the city with the index 23 
- Information of the city 23 -
-> Number of current patients : 159
-> Has recovered : 512
-> Was dead : 0

Query ? 4 9 :
The answer is the city with the index 13 
- Information of the city 13 -
-> Number of current patients : 127
-> Has recovered : 0
-> Was dead : 111

Query ? 7 :
The city with the highest death is city with the index 4
Quantity: 218 people

Query ? 6 :
The city with the highest recovery is city with the index 23
Quantity: 512 people

Query ? 4 2 :
The answer is the city with the index 9 
- Information of the city 9 -
-> Number of current patients : 3
-> Has recovered : 6
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 23
Quantity: 512 people

Query ? 4 9 :
The answer is the city with the index 13 
- Information of the city 13 -
-> Number of current patients : 127
-> Has recovered : 0
-> Was dead : 111

Query ? 4 7 :
The answer is the city with the index 12 
- Information of the city 12 -
-> Number of current patients : 73
-> Has recovered : 21
-> Was dead : 0

Query ? 4 4 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 34
-> Has recovered : 253
-> Was dead : 134

Query ? 4 10 :
The answer is the city with the index 32 
- Information of the city 32 -
-> Number of current patients : 159
-> Has recovered : 0
-> Was dead : 0

Query ? 4 18 :
The answer is the city with the index 25 
- Information of the city 25 -
-> Number of current patients : 491
-> Has recovered : 0
-> Was dead : 0

Query ? 4 8 :
The answer is the city with the index 44 
- Information of the city 44 -
-> Number of current patients : 77
-> Has recovered : 0
-> Was dead : 0

Query ? 4 4 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 34
-> Has recovered : 253
-> Was dead : 134

Query ? 4 14 :
The answer is the city with the index 30 
- Information of the city 30 -
-> Number of current patients : 207
-> Has recovered : 0
-> Was dead : 0

Query ? 4 9 :
The answer is the city with the index 44 
- Information of the city 44 -
-> Number of current patients : 77
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 9 
- Information of the city 9 -
-> Number of current patients : 3
-> Has recovered : 6
-> Was dead : 0

Query ? 4 18 :
The answer is the city with the index 25 
- Information of the city 25 -
-> Number of current patients : 491
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 4
Quantity: 218 people

Query ? 7 :
The city with the highest death is city with the index 4
Quantity: 218 people

Query ? 4 7 :
The answer is the city with the index 4 
- Information of the city 4 -
-> Number of current patients : 64
-> Has recovered : 53
-> Was dead : 218

Query ? 7 :
The city with the highest death is city with the index 4
Quantity: 218 people

Query ? 4 14 :
The answer is the city with the index 30 
- Information of the city 30 -
-> Number of current patients : 207
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 4
Quantity: 218 people

Query ? 4 16 :
The answer is the city with the index 20 
- Information of the city 20 -
-> Number of current patients : 335
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 4
Quantity: 218 people

Query ? 6 :
The city with the highest recovery is city with the index 23
Quantity: 512 people

Query ? 7 :
The city with the highest death is city with the index 4
Quantity: 218 people

Query ? 4 10 :
The answer is the city with the index 26 
- Information of the city 26 -
-> Number of current patients : 87
-> Has recovered : 198
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 27 
- Information of the city 27 -
-> Number of current patients : 6
-> Has recovered : 192
-> Was dead : 40

Query ? 4 5 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 34
-> Has recovered : 253
-> Was dead : 134

Query ? 7 :
The city with the highest death is city with the index 4
Quantity: 218 people

